ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts

```diff
@@ -4,15 +4,15 @@ import { bcsSerializeBool } from "../../bcs";
 import { FaucetClient } from "../../plugins/faucet_client";
 import { IndexerClient } from "../../providers/indexer";
 import { TokenClient } from "../../plugins/token_client";
-import { API_TOKEN, longTestTimeout } from "../unit/test_helper.test";
+import { FAUCET_AUTH_TOKEN, longTestTimeout } from "../unit/test_helper.test";
```

Comment:
thanks for fixing it! maybe worth getting it to test_helper file, something like `getTestnetFaucet()` https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/tests/unit/test_helper.test.ts

Reply:
yes, a new `getTestnetFaucet()`. So all tests run against local node, but indexer tests runs against testnet (this is b/c spinning up a local indexer is almost a headache).  `getFaucetClient()` uses local node endpoints and we want to keep it like that. 

BUT feel free to leave it for a different/second PR